akamai/src/main/java/nl/nn/adapterframework/extensions/akamai/NetStorageSender.java

```diff
@@ -242,12 +240,10 @@ public Message extractResult(HttpResponseHandler responseHandler, PipeLineSessio
 				String dateString = responseHandler.getHeader("Date");
 				if(!StringUtils.isEmpty(dateString)) {
 					Date currentDate = new Date();
-					DateFormat format = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss zzz", Locale.ENGLISH);
 					long responseDate = 0;
 
 					try {
-						Date date = format.parse(dateString);
-						responseDate = date.getTime();
+						responseDate = DateUtils.parseToInstant(dateString, "EEE, dd MMM yyyy HH:mm:ss zzz").toEpochMilli();
```

Comment:
Je bent nu heel veel bestanden aan het passen, probeer ook te kijken waar de datum voor is. In dit geval puur om te kijken of iets binnen een tijdsbestek valt, het wordt er niet beter op zo, kan dit anders?